recipes/yaclib/all/conanfile.py

```diff
@@ -0,0 +1,159 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps
+from conan.tools.scm import Version
+from conan.tools.files import copy, patch, get
+from conan.errors import ConanInvalidConfiguration
+from functools import lru_cache
+import os
+
+required_conan_version = ">=1.47.0"
+
+
+class YACLibConan(ConanFile):
+    name = 'yaclib'
+    description = 'lightweight C++ library for concurrent and parallel task execution'
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'https://github.com/YACLib/YACLib'
+    license = 'MIT'
+    topics = 'c++', 'async', 'parallel', 'concurrency'
+    settings = 'os', 'arch', 'compiler', 'build_type'
+    exports_sources = 'patches/**'
+
+    _yaclib_flags = {
+        'warn': [True, False],
+        'asan': [True, False],
+        'tsan': [True, False],
+        'ubsan': [True, False],
+        'lsan': [True, False],
+        'memsan': [True, False],
+        'coverage': [True, False],
+        'coro': [True, False],
+        'disable_futex': [True, False],
+        'disable_unsafe_futex': [True, False],
+        'disable_symmetric_transfer': [True, False],
+        'disable_final_suspend_transfer': [True, False],
+    }
+
+    _yaclib_faults = {
+        'thread_fault': [True, False],
+        'fiber_fault': [True, False],
+    }
+
+    options = {
+        'fPIC': [True, False],
+        **_yaclib_flags,
+        **_yaclib_faults,
+    }
+
+    default_options = {
+        'fPIC': True,
+        **{k: False for k in _yaclib_flags.keys()},
+        **{k: False for k in _yaclib_faults.keys()},
+    }
+
+    _cmake_definitions = {}
+
+    @property
+    def _source_subfolder(self):
+        return 'source_subfolder'
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            'gcc': '7',
+            'Visual Studio': '14.20',
+            'clang': '8',
+            'apple-clang': '12',
+        }
+
+    def _patch_sources(self):
+        for p in self.conan_data.get("patches", {}).get(self.version, []):
+            patch(self, **p)
+
+    @lru_cache(1)
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        defs = {}
+        if not self.conf.get(
```

Comment:
These should always be off in CCI recipes, the configuration is for users to have more workflow, it's not needed here

Every recipe would need this and need to be tested which is very expensive to do 🙊

Reply:
Removed _configure_cmake